src/components/App/App.tsx

```diff
@@ -35,20 +47,26 @@ const App:React.FC = () =>{
   return (
     <QuestionContext.Provider value={questions}>
       <AnswerContext.Provider value={answers}>
+        <ReportContext.Provider value={report}>
         <NavBar/>
-        <Switch>
-          <Route exact path="/home" component={Home}/>
-          <Route exact path="/journey" component={Journey}/>
-          <Route exact path="/survey" component={Survey}/>
-          <Route exact path="/question" component={()=><Question
-             updateAllAnswers={updateAllAnswers}/>}/>
-          <Route exact path="/generate_report" component={GenerateReport}/>
-          <Route exact path="/submit" GenerateReport/>
-          <Route exact path="/report" component={Report} />
-          <Route path='/*' component={Error}/>
-        </Switch>
```

Comment:
Just a note, we may need to adjust this pathing to make home our default page. We will have to test if it currently loads to the 404 page or home on launch